### PR TITLE
add iiif_presentation to formats that are always recreated, bypassing…

### PR DESCRIPTION
**Pull request by mksndz**
… 'freshness' check

### CODE REVIEW

File: application.rb [hunk 1]
```diff
@@ -486,6 +486,10 @@ def still_fresh?(bib_id, format)
   end
 end
 
+def always_consider_fresh?(format)
```
Comment (cgalarza) [inline application.rb:489]: `always_consider_stale`?

Comment (mksndz) [reply application.rb]: oops, fixed by renaming everything

**Comment by cgalarza** [comment]
@mksndz I think we should add some tests before we merge this. We also may need to rebase. Do you have bandwidth to work on this? If not, I'll take a stab at it.

**Comment by mksndz** [comment]
@cgalarza sure, i'll be happy to rebase and add tests here

**Comment by mksndz** [comment]
I'm unhappy with how I've done things here - I've started over @ https://github.com/upenn-libraries/marmite/tree/bypass_freshness_for_iiif_2

**Pull request closed without merging**